test/test_tensor_data.py

```diff
@@ -31,6 +31,14 @@ def test_data_nested(self):
     assert dat[0, 0] == 1
     assert dat[1, 1] == 4
 
+  def test_data_const(self):
+    a = Tensor(3, dtype=dtypes.int32)
+    dat = a.data()
+    assert dat.format == "i"
+    assert dat.itemsize == 4
+    assert dat.tolist() == 3
```

Comment:
slightly weird that `.tolist()` returns a non-list, will let @geohot decide

Reply:
It's fine, it matches PyTorch